depends/packages/native_clang.mk

```diff
@@ -16,10 +16,13 @@ endef
 define $(package)_stage_cmds
   mkdir -p $($(package)_staging_prefix_dir)/lib/clang/$($(package)_version)/include && \
   mkdir -p $($(package)_staging_prefix_dir)/bin && \
+  mkdir -p $($(package)_staging_prefix_dir)/include/llvm-c && \
   cp bin/clang $($(package)_staging_prefix_dir)/bin/ && \
   cp -P bin/clang++ $($(package)_staging_prefix_dir)/bin/ && \
   cp bin/dsymutil $($(package)_staging_prefix_dir)/bin/$(host)-dsymutil && \
   cp bin/llvm-config $($(package)_staging_prefix_dir)/bin/ && \
+  cp include/llvm-c/ExternC.h $($(package)_staging_prefix_dir)/include/llvm-c && \
```

Comment:
9b60690b94ca766f8039a7a7a20dda5fecd176ab

Why this line has been added?

FWIW, I've managed to build for both darwin hosts with `LTO=1` in Guix using the #25391 branch without this line.

Reply:
Because it's included by the `lto.h` header.

Reply:
It's not completely suprising builds succeed because the file is only copied over (the existing ExternC.h) inside cctools if it exists.